md/BonitaBPM_platform_setup.md

```diff
@@ -1,34 +1,32 @@
 
 # Platform configuration
 
-In this section you will learn what the Platform setup tool is and how to use it to initialize and configure the Bonita BPM Platform.
+In this section you will learn what the Platform setup tool is and how to use it to configure the Bonita BPM Platform, manage your licenses (for Subscription users), and for special use cases, initialize the bundle.
```

Comment:
`for special use cases, initialize the bundle` => I don't understand what you mean...